src/web/components/icon/deltadifferenceicon.js

```diff
@@ -0,0 +1,26 @@
+/* Copyright (C) 2018-2022 Greenbone AG
```

Comment:
If that is a new file it should be
```suggestion
/* Copyright (C) 2023 Greenbone AG
```